src/drafter.cc

```diff
@@ -83,7 +127,7 @@ DRAFTER_API drafter_error drafter_parse_blueprint(
 
     sc::BlueprintParserOptions scOptions = sc::ExportSourcemapOption;
 
-    if (parse_opts.requireBlueprintName) {
+    if (parse_opts->requireBlueprintName) {
```

Comment:
If `parse_opts` is NULL we'd crash here. Should we add a check above and provide an error? (or support `parse_opts` being NULL an `requireBlueprintName` goes to default value (false)

Reply:
Allowing null options may be nicer as a user of the library, especially when getting started.

Reply:
Yes we can probably just use the default values on nullptr.